ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -98,16 +102,21 @@ class TabBarItemView: UIControl {
         scalesLargeContentImage = true
 
         NSLayoutConstraint.activate([
-			container.centerXAnchor.constraint(equalTo: centerXAnchor),
-			container.centerYAnchor.constraint(equalTo: centerYAnchor),
-			container.widthAnchor.constraint(lessThanOrEqualTo: widthAnchor)
-		])
+            container.centerXAnchor.constraint(equalTo: centerXAnchor),
+            container.centerYAnchor.constraint(equalTo: centerYAnchor),
+            container.widthAnchor.constraint(lessThanOrEqualTo: widthAnchor)
+    ])
 
         NotificationCenter.default.addObserver(self,
                                                selector: #selector(badgeValueDidChange),
```

Comment:
if badgeValuedidchange getcalled, when there was already isunreaddot visible, then we need to have the logic to hide the dot, and redrawe the badgevalue.

Reply:
This currently works as intended and as you have described.

Reply:
where is that code?

Reply:
Refactored a bunch of code. I've confirmed that the subview is removed when the unread dot is not visible.